packages/flutter/lib/src/material/input_date_picker_form_field.dart

```diff
@@ -125,6 +126,11 @@ class InputDatePickerFormField extends StatefulWidget {
   /// string. For example, 'Month, Day, Year' for en_US.
   final String? fieldLabelText;
 
+  /// The keyboard type of the [TextField].
+  ///
+  /// If this is null, it will default to [TextInputType.datetime]
```

Comment:
Nit: This comment is the same as in date_picker.dart, so you could use a template and macro to reuse it (see autofocus just below this using a macro).